main.nf

```diff
@@ -562,6 +605,7 @@ if(params.containsKey("bidirectional") && params.bidirectional){
 
 process join_counts {
     input:
+        file("raw.csv") from raw_counts_concat
```

Comment:
Are cutadapt counts making it into here? i don't see a reference to ``cutadapt_counts_concat``, which I assume would hold the input file.

Reply:
Okay not ideal, but after running out of patience trying to figure out how to concat then expand a non-deterministic list of counts_concat channels I went with the approach seen [here](https://github.com/nhoffman/dada2-nf/pull/58/commits/51dd38bc82fb66184c51440e33a25b11b56d848e). Basically defines an empty `cutadapt_counts_concat` channel and then tests channel content to set the join command. Happy to have this adjusted if either @nhoffman or @crosenth knows how to better achieve.